Riot/Utils/RiotDesignValues.m

```diff
@@ -109,6 +113,8 @@ + (void)load
     kRiotColorLightGreen = UIColorFromRGB(0x50e2c2);
     kRiotColorLightOrange = UIColorFromRGB(0xf4c371);
     
+    kRiotKeyboard = UIKeyboardAppearanceLight;
```

Comment:
This initialisation is not required here because `userInterfaceThemeDidChange` will be called.